02Lesson/02Homework/index.html

```diff
@@ -3,24 +3,24 @@
 
 <head>
   <title>Git</title>
-  <link href="css/style.css" rel="stylesheet"/>
+  <link type="text/css" rel="stylesheet" href="stylesheet.css"/>
 </head>
 
 <body>
-  <div id="header">
-    <div id="nav">
+  <header>
+    <nav>
       <span><a href="#">Home</a></span>
       <span><a href="#">How does it work</a></span>
       <span><a href="#">Contact Us</a></span>
-    </div>
-  </div>
-  <div id="main">
-    <div class="section">
+    </nav>
+  </header>
+  <main>
+    <section>
       <img src="http://git-scm.com//images/logo@2x.png" alt="git logo"/>
       <h2>Welcome to Git</h2>
       <p>Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency. </p>
-    </div>
-    <div class="section">
+    </section>
+  </main>
       <div class="article">

```

Comment:
Indentation: After closing `main` tag, the next opening element must be on the same indentation level